### ISSUE TITLE
Stuck in exporting settings

### ISSUE THREAD

**Issue by aacebedo** [golf1052/code-sync#37]
VSCode version:
1.39.1
CodeSync version:
2.6.0
CodeSync log:
```
settings.ts
	2019-10-13T12:28:14.984Z - Creating settings with internal settings.json path and external extensions.json path: /home/aacebedo/.vscode/extensions/golf1052.code-sync-2.6.0/settings.json, extensions.json.
helpers.ts
	2019-10-13T12:28:15.188Z - Found code version: 1.39.1
88f15d17dca836346e787762685a40bb5cce75a8
x64
.
cs.ts
	2019-10-13T12:28:15.189Z - Checking settings
	2019-10-13T12:28:15.199Z - Setting external extensions path as /tmp/test/extensions.json.
	2019-10-13T12:28:15.199Z - Done checking settings.
	2019-10-13T12:28:15.199Z - External sync directory: /tmp/test
	2019-10-13T12:28:15.203Z - Importing settings
	2019-10-13T12:28:15.203Z - Failed to import settings. Could not find settings.json at /tmp/test/settings.json. Giving up.
	2019-10-13T12:28:15.203Z - Importing keybindings
	2019-10-13T12:28:15.203Z - Failed to import keybindings. Could not find keybindings.json at /tmp/test/keybindings.json. Giving up.
	2019-10-13T12:28:15.204Z - Importing snippets
	2019-10-13T12:28:15.204Z - Failed to import snippets. Could not find snippets directory at /tmp/test/snippets. Giving up.
	2019-10-13T12:28:15.204Z - Importing extensions
	2019-10-13T12:28:15.211Z - Finished importing extensions.
	2019-10-13T12:28:15.211Z - Exporting extensions
	2019-10-13T12:28:15.212Z - Finished exporting extensions

```

Issue: 
When launching the export settings command, the status displays "Exporting settings" and gets stuck. Nothing happens after that and I don't see any errors in any logs.

I am using vscode on Linux.

**Comment by golf1052** [comment]
I've released version 2.6.1 which adds more logging when exporting settings. Try updating to that version and pasting your log output again.

What I think is happening is that CodeSync can't find your VSCode settings file which on Linux should be at `~/.config/Code/User/settings.json` [(see documentation)](https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations)

**Comment by golf1052** [comment]
Cleaning up old issue.
